spec/support/multitenancy_metadata.rb

```diff
@@ -3,16 +3,38 @@
   # might not actually be used in all examples where it's included.
   config.after do
     example = RSpec.current_example
-    if example.metadata[:multitenant]
+    if example.metadata[:multitenant] || example.metadata[:singletenant]
       allow(Settings.multitenancy).to receive(:enabled).and_call_original
       Rails.application.reload_routes!
     end
   end
+
+  # There are 3 optional flags available to a test block.  Only ONE will be active
+  # at any given time.  They are (with areas of likely use):
+  #   :multitenant  - general case default, only needs to be explicit for types described below
```

Comment:
Can we collapse this to a single setting? `tenancy: :single`, `tenancy: :multi`, `tenancy: :fake`?